### ISSUE TITLE
Can't overfit on a single piece of training data

### ISSUE THREAD

**Issue by Chris-Pedersen** [Chris-Pedersen/LearnableWavelets#3]
I would expect the loss to converge really fast and overfit when running the model (either camels CNN or a scattering network) on a single training point, but for some reason this doesn't happen. Will likely point to a bug in the model somewhere.
